tests/testutils.py

```diff
@@ -36,3 +37,25 @@ def normalize(content):
         out = content
 
     return fix_lineseps(out)
+
+
+def sample_unformatted() -> str:
+    """Provide a small unformatted source sample for general testing."""
+    return """\
+# A sample file which needs some formatting.
+
+# NOTE: More extensive tests should go into unit tests or separate files in `tests/samples`.
+
+global    foo=1   +2    ;
+"""
+
+
+def sample_formatted() -> str:
+    """Provide a small formatted source sample for general testing."""
+    return """\
+# A sample file which needs some formatting.
+
+# NOTE: More extensive tests should go into unit tests or separate files in `tests/samples`.
+
+global foo = 1 + 2;
+"""
```

Comment:
Hmm, doesn't look super nice and a bit noisy :-/

Might look a bit more concise as constants? And maybe removing the `# NOTE` comment and possibly the other comment, too (and put one atop outside, saying "do not add more samples here, instead use `tests/samples` or so).

Reply:
I made these constants. I dropped the note since it seems unlikely people to add more stuff here.